src/components/Trade/hooks/useSwapper/useSwapper.ts

```diff
@@ -155,7 +184,11 @@ export const useSwapper = () => {
     buyAsset: Asset
     amount: string
   }): Promise<Quote<ChainTypes, SwapperType> | undefined> => {
-    const swapper = swapperManager.getSwapper(bestSwapperType)
+    const swapper = await swapperManager.getBestSwapper({
+      buyAssetId: buyAsset.caip19,
```

Comment:
please swap all these `caip19`s to `assetId`s

Reply:
We actually cant do this yet. The type exists but asset.assetId is undefined so we have to keep using asset.caip19

Reply:
now you can with types@3.0